io/disk/disktest.py

```diff
@@ -64,37 +64,38 @@ def setUp(self):
         """
         smm = SoftwareManager()
         if not smm.check_installed("gcc") and not smm.install("gcc"):
-            self.cancel('Gcc is needed for the test to be run')
+            self.cancel("Gcc is needed for the test to be run")
         # Log of all the disktest processes
-        self.disk_log = os.path.abspath(os.path.join(self.outputdir,
-                                                     "log.txt"))
+        self.disk_log = os.path.abspath(os.path.join(self.outputdir, "log.txt"))
         self.fs_create = False
-        self.raid_needed = self.params.get('raid', default=False)
+        self.raid_needed = self.params.get("raid", default=False)
         self.raid_create = False
-        self.disk = self.params.get('disk', default=None)
-        self.dir = self.params.get('dir', default=None)
-        self.fstype = self.params.get('fs', default='ext4')
-        self.raid_name = '/dev/md/sraid'
+        self.disk = self.params.get("disk", default=None)
+        self.dir = self.params.get("dir", default="/mnt")
```

Comment:
@AYUSHJAIN951 I think this may not be the right think to give /mnt as default dir. every one doesn't uses the /mnt for their test and some time it may get messup with OS disk space like for fiotest for creating the temp files on test runs.
So I advise to keep it as "None" only.

Reply:
yes i agree with @Naresh-ibm in your use case if you are looking to use upstream repo please have yam file and driver your scenario using that

Reply:
Thanks for these recommendations rather than "/mnt", we can have self.workdir as the self.dir in case when self.disk is provided and self.dir is not and also if both are not provided by the user.